src/test/strtol.cc

```diff
@@ -257,6 +257,21 @@ TEST(StrictIECCast, Error) {
     (void)strict_iec_cast<int>("1T", &err);
     ASSERT_NE(err, "");
   }
+  {
+    std::string err;
+    (void)strict_iec_cast<int64_t>("1GT", &err);
+    ASSERT_NE(err, "");
+  }
+  {
+    std::string err;
+    (void)strict_iec_cast<int64_t>("1TG", &err);
+    ASSERT_NE(err, "");
+  }
+  {
+    std::string err;
+    (void)strict_iec_cast<int64_t>("1KD", &err);
+    ASSERT_NE(err, "");
+  }
```

Comment:
Please add tests for `1KiB`/etc.

Reply:
Updated.

Reply:
It occurred to me now that this test case is for checking erroneous units. Please move these tests to

`TEST(IECStrToLL, WithUnits)`

Reply:
Yup, you are right!